packages/radio-group/src/Radio/Radio.tsx

```diff
@@ -90,7 +96,7 @@ function Radio({
           id={id}
           name={name}
           type="radio"
-          onChange={onChange}
+          onChange={handleChange}
           value={value}
           aria-checked={checked}
           aria-disabled={disabled}
```

Comment:
@bruugey should we just be using the disabled attribute in this case?

Reply:
Trying to understand why we don't do this currently..

Reply:
Maybe bc folks wanted to put a tooltip on these elements?

Reply:
I think it's just an oversight, because a tool tip would show up on the label, not the input element itself

Reply:
Not with a RadioBoxGroup or Button - so perhaps for consistency?

Reply:
the focus is so that if it's a tooltip trigger it can still be used as such, but this feels/looks bad